src/soep_preparation/clean_variables/pl.py

```diff
@@ -31,6 +32,47 @@ def _private_rente_beitrag_jahr(
     return out.where(eingezahlt.astype("bool[pyarrow]"), 0)
 
 
+def _private_rente_beitrag_monatlich(data: pd.DataFrame) -> pd.Series:
+    private_rente_beitrag_m = pd.Series(
```

Comment:
```suggestion
    combined_years = pd.Series(
```
?

If possible, use an `==` condition, though -- always cumbersome to work with double negations (the other coming from `where`).